extensions/wikia/WikiaRSS/WikiaRssHelper.class.php

```diff
@@ -7,7 +7,7 @@ class WikiaRssHelper {
 	 *
```

Comment:
Changes in this file related to [SOC-522](https://wikia-inc.atlassian.net/browse/SOC-522)